services/browser-service/src/browser_service/browser_manager.py

```
@@ -23,25 +49,157 @@ async def start(self):
+
+                for session_id, session_info in self._sessions.items():
+                    if now - session_info.last_used > self._session_ttl:
+                        expired.append(session_id)
+
+                for session_id in expired:
+                    logger.info(f"Cleaning up expired session: {session_id}")
+                    await self.close_session(session_id)
+
+            except asyncio.CancelledError:
+                break
+            except Exception as e:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)